File: design-patterns/JavaDesignPatterns/src/com/bigboxcode/designpattern/facade/transport/Weather.java
// Weather.java
// All function implementations are dummy implementations, just to demonstrate Facade

package com.bigboxcode.designpattern.facade.transport;

public class Weather {

    public void getWeatherInfo(double lat, double lng) {
        System.out.println("Temperature: 20.7");
        System.out.println("Precipitation: 1%");
        System.out.println("Humidity: 73%");
        System.out.println("Wind: 8 km/h");
    }
}

File: design-patterns/php/facade/travelplan/Weather.php
<?php
// Weather.php

namespace BigBoxCode\DesignPattern\Facade\TravelPlan;

class Weather {
    public function getWeatherInfo(float $lat, float $lng): void {
        echo "Temperature: 20.7\n";
        echo "Precipitation: 1%\n";
        echo "Humidity: 73%\n";
        echo "Wind: 8 km/h\n";
    }
}